wallet/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
 
     defaultConfig {
         applicationId "com.android.identity_credential.wallet"
-        minSdk 27
+        minSdk 28
```

Comment:
And reason for this?

Reply:
moved up an API level in order to use `android.content.Context#getMainExecutor` when building the `NfcEngagementHelper` in NfcEngagementHandler.kt